build.cmd

```diff
@@ -51,7 +51,6 @@ set __SkipConfigure=
 set __SkipMscorlibBuild=
 set __SkipNativeBuild=
 set __SkipTestBuild=
-set __DoCrossgen=
```

Comment:
But this change will break netci.groovy unless you remove the uses of 'docrossgen' from that file as part of the change